build.sbt

```diff
@@ -86,6 +89,30 @@ lazy val commonSettings = Seq(
     else Some("releases" at nexus + "service/local/staging/deploy/maven2")
   })
 
+lazy val micrositesSettings = Seq(
+  micrositeName := "pureconfig",
+  micrositeDescription := "A boilerplate-free library for loading configuration files ",
+  micrositeAuthor := "pureconfig authors",
+  micrositeHomepage := "https://pureconfig.github.io/",
+  //micrositeBaseUrl := "pureconfig", // keep this empty to not have a base URL
+  //micrositeDocumentationUrl := "api/", // don't have yet
```

Comment:
How do you plan to provide a documentation URL? It would be nice to have a link to the documentation in the top bar of the homepage, since the "Documentation" section below is a bit hard to find. Wouldn't a natural way to do that be to scope all Markdown files inside a `docs/src/main/tut/docs` folder and use `micrositeDocumentationUrl  := "docs/"` here?

Reply:
The URL can be the first page of the documentation, I just forgot to set it. I pushed that fixes that and now clicking on `Documentation` should open `setup-pureconfig.html`.

Reply:
Ok, it works well now. Still, it seems strange to me to define an arbitrary page of the documentation as "main page"...

In general, it seems that we are mixing the homepage part of the website (index.md) with the documentation (all other markdown files). They are presented with different styles and, conceptually, it would make more sense to distinguish them by moving all documentation to https://pureconfig.github.io/docs/, for example. That's the approach used by sbt-microsites ([home](https://47deg.github.io/sbt-microsites), [docs](https://47deg.github.io/sbt-microsites/docs)).

If we organize the documents that way, `micrositeDocumentationUrl` takes the natural value of `docs/`, which is the root of the documentation and always points to the first page, which is not hardcoded.

Reply:
https://github.com/pureconfig/pureconfig/pull/319/commits/35faf4ece7ad99b8d03adf7a4f3fe9e95b7010a0